hw3/index.js

```diff
@@ -0,0 +1,31 @@
+// import task 1
+const multiplication = require("./multiplication");
+// import task 2
+const cardCreating = require("./card");
+// import task 3
+const car = require("./car");
+
+// usage task 1
+console.log("====================TASK 1 ==================");
+console.log(multiplication.multi(3));
+console.log(multiplication.multi(5));
+console.log(multiplication.multi(2));
```

Comment:
this usage is wrong. In task example it was
``` js
const mul3 = mul(3);
const mul15 = mul3(5) ;// 15
console.log(mul15(2)); // 30
```
Bug get 
mul3 is not a function

Reply:
so far i have done with using valueOf, after homework5 , if i can, i return to this tusk.